### DISCUSSION TITLE
[feature suggestion] create utils to limit media size and dimensions to telegram sending limits

### DISCUSSION THREAD

**Discussion by batmanscode** [eternnoir/pyTelegramBotAPI#1829] [Ideas]
Please answer these questions before submitting your issue. Thanks!

1. What version of pyTelegramBotAPI are you using?
N/A
2. What OS are you using?
N/A
3. What version of python are you using?
N/A

This is a feature suggestion to include an optional utility or `smart_send_media()` option that checks the size and dimensions of the media being sent and compresses/resizes if it's exceeding telegram limits.

**Comment by coder2020official** [comment]
I'll think about it, thanks for suggestion

**Comment by batmanscode** [comment]
> Please answer these questions before submitting your issue. Thanks!
> 
> 1. What version of pyTelegramBotAPI are you using?
> N/A
> 2. What OS are you using?
> N/A
> 3. What version of python are you using?
> N/A
> 
> This is a feature suggestion to include an optional utility or `smart_send_media()` option that checks the size and dimensions of the media being sent and compresses/resizes if it's exceeding telegram limits.

Update:

It would be great if `smart_send_media()` also automatically sends multiple files as albums from a list of local paths or URLs
